scripts/run-nat-node.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+function echoerr() { echo "$@" 1>&2; }
+
+if [ $(id -u) -ne 0 ]; then 
+  echoerr "ERR: Please run as root!"
+  exit 1 
+fi
+
+
+declare rundir="/var/hoprd"
```

Comment:
Since this directory is bound into the container, I'd rather have this configurable or with a more descriptive name.